less/shared.less

```diff
@@ -263,10 +287,7 @@ input[type="email"] {
 
 input[type="email"] {
   padding-right: 33px;
-}
-input[name="lastname"],
-input[name="zip"] {
-  padding-left: 14px;
+  margin-top: 3px;
```

Comment:
That gradient shouldn't be there at all. It's only happening for chrome and safari on a mac, not seeing it in windows or linux.